tests/lib/bash/test_header

```diff
@@ -550,6 +550,12 @@ skip_all() {
     exit
 }
 
+skip_darwin() {
+    if [[ "$OSTYPE" == "darwin"* ]]; then
+        skip_all "Skipping (Mac OS/Darwin) $*"
```

Comment:
> I'll take your word that skip_darwin does what it says on the tin

It uses the `OSTYPE` environment variable, MacOS and Darwin derivatives all identify as `darwin`.